docs/meshcloud.customer.md

```diff
@@ -16,11 +16,11 @@ and any additional [metadata specific to your organization](meshstack.metadata-t
 
 ## Managing your meshCustomer
 
-Every aspect of your meshCustomer can be managed in the so-called [customer control plane](./meshcloud.customer.md#managing-your-meshcustomer). This customer control plane looks like the screenshot below.
+Every aspect of your meshCustomer can be managed in the so-called [customer control plane](./meshcloud.customer.md#managing-your-meshcustomer). The customer control plane is the highest control plane. From that level, you can navigate to various control planes like the [project control plane](./meshcloud.project-resources.md#project-control-plane) or the [tenant control plane](./meshcloud.project-resources.md#tenant-control-plane). Each control plane has a similar scheme. Depending on your permissions within the customer and the meshStack configuration, you will have access to different to tabs like **Settings** or **Financials**. The customer control plane below shows the control plane from the perspective of a Customer Admin.
```

Comment:
I think it would support new users when the we describe shortly on a high level what the deferent control planes are for

Reply:
d: does it fit under managing meshCustomer?
Or should i do it in the overview?

Reply:
overview would be alright

Reply:
![image](https://user-images.githubusercontent.com/64793818/148556268-fb46ff5c-712a-4b40-8e2c-85210b008421.png)